annotations/views.py

```diff
@@ -266,7 +308,22 @@ def get_object(self, queryset=None):
         return fragment
 
 
-class FragmentEdit(SelectSegmentMixin, LoginRequiredMixin, generic.UpdateView):
+class FragmentRevisionWithCommentMixin(RevisionWithCommentMixin):
+    def find_in_enum(self, key, enum):
```

Comment:
There's no standard Python/Django method for this? :-)

Reply:
The thing is, those constants in Fragment aren't really python enums, but tuples of tuples. I changed it to something slightly better.